### PR TITLE
Updated to use the api v9 for toggl

### PR DESCRIPTION
**Pull request by daren-thomas**
(the toggl reports api seems to still work, so I didn't change anything there...)

### CODE REVIEW

File: src/Toggl2Vertec/Properties/launchSettings.json [hunk 1]
```diff
@@ -3,6 +3,18 @@
     "Toggl2Vertec": {
       "commandName": "Project",
       "commandLineArgs": "clear --debug"
+    },
+    "check": {
```
Comment (lord-executor) [inline src/Toggl2Vertec/Properties/launchSettings.json:7]: Now I know who uses _that_ functionality.

File: src/Toggl2Vertec/Toggl/TogglAccessCheck.cs [hunk 2]
```diff
@@ -15,11 +15,11 @@ public TogglAccessCheck(TogglClient client)
 
         public override bool Check(ICliLogger logger)
         {
-            logger.LogPartial(logger.CreateText($"Checking Toggl API access (https://api.track.toggl.com/api/v8/me): "));
+            logger.LogPartial(logger.CreateText("Checking Toggl API access (https://api.track.toggl.com/api/v9/me): "));
             try
             {
                 var profile = _client.FetchProfileDetails();
-                if (profile.GetProperty("data").GetProperty("id").GetInt32() <= 0)
+                if (profile.GetProperty("id").GetInt32() <= 0)
```
Comment (lord-executor) [inline src/Toggl2Vertec/Toggl/TogglAccessCheck.cs:22]: I see. _Massive_ restructuring of the data structure. I'm sure that was worth it for Toggl 😝

File: src/Toggl2Vertec/Properties/launchSettings.json (hunk 1 above)
Comment (daren-thomas) [reply src/Toggl2Vertec/Properties/launchSettings.json:7]: rider was doing it's best to steer me in that direction _shrug_...

File: src/Toggl2Vertec/Toggl/TogglAccessCheck.cs (hunk 2 above)
Comment (daren-thomas) [reply src/Toggl2Vertec/Toggl/TogglAccessCheck.cs:22]: version nine! it's... bigger than eight! 12.5% increase in API version! just imagine the exponential potential of a 12.5% growth rate!! in two years time, the whole universe could be paper clips and we'd _still_ be increasing the version!!!

**Pull request merged**